pytorch/pytorch-native/src/ai_djl_pytorch_jni_PyTorchLibrary_inference.cc

```diff
@@ -23,12 +25,72 @@ struct JITCallGuard {
   torch::NoGradGuard no_grad;
 };
 
+inline std::vector<std::string> split(std::string& str, const std::string& delimiter = ".") {
+  std::vector<std::string> vec;
+  size_t pos;
+  std::string token;
+  while ((pos = str.find(delimiter)) != std::string::npos) {
+    token = str.substr(0, pos);
+    vec.emplace_back(token);
+    str.erase(0, pos + delimiter.length());
+  }
+  vec.emplace_back(str);
+  return std::move(vec);
+}
+
+inline bool is_array(std::string& name) { return std::regex_match(name, std::regex(R"(\w+\[\])")); }
+
+inline bool is_dict(std::string& name) { return name.find('.') != std::string::npos; }
+
+inline std::vector<torch::jit::IValue> GetInputs(JNIEnv* env, jobjectArray tensor_ptrs, jobjectArray jnames) {
```

Comment:
Similar here, try to split this large function into the types, if the input type is x, then do x operation